torchaudio/models/_wavernn.py

```diff
@@ -192,3 +193,131 @@ def forward(self, specgram: Tensor) -> Tensor:
         upsampling_output = upsampling_output.squeeze(1)[:, :, self.indent:-self.indent]
 
         return upsampling_output, resnet_output
+
+
+class _WaveRNN(nn.Module):
+    r"""WaveRNN model based on "Efficient Neural Audio Synthesis".
+
+    The paper link is `<https://arxiv.org/pdf/1802.08435.pdf>`_. The input channels of waveform
```

Comment:
nit:  For more info see the paper `Efficient Neural Audio Synthesis <https://arxiv.org/pdf/1802.08435.pdf>`_

Reply:
Given the feedback in [comment](https://github.com/pytorch/audio/pull/749#issuecomment-653630756), I suggest we make it more explicit that this version is not the original.

```
"""
WaveRNN model based on the implementation from `fatchord <https://github.com/fatchord/WaveRNN>`_ .

The original implementation was introduced in `"Efficient Neural Audio Synthesis" <https://arxiv.org/pdf/1802.08435.pdf>`_.

The input channels of waveform and spectrogram have to be 1. The product of `upsample_scales` must equal `hop_length`.
"""
```

If it doesn't make the first line too long, I would merge the first and second paragraph. Thoughts?

Reply:
Another option is to add a flag in WaveRNN init, say `version` that can only take the value `fatchord` for now, but could eventually also take the value `deepmind` later if/when implemented. Thoughts?

Reply:
(Thanks for the feedback in [comment](https://github.com/pytorch/audio/pull/749#issuecomment-653630756) @PetrochukM :))